src/xrpld/app/tx/detail/DepositPreauth.cpp

```diff
@@ -94,45 +92,13 @@ DepositPreauth::preflight(PreflightContext const& ctx)
     }
     else
     {
-        STArray const& arr(ctx.tx.getFieldArray(
-            authArrPresent ? sfAuthorizeCredentials
-                           : sfUnauthorizeCredentials));
-        if (arr.empty() || (arr.size() > maxCredentialsArraySize))
-        {
-            JLOG(ctx.j.trace()) << "Malformed transaction: "
```

Comment:
Please port over the log lines to the helper functions as well

Reply:
added logs